platforms/dotnet/versions/buster/versionsToBuild.txt

```diff
@@ -59,6 +59,7 @@
 6.0.410, 8c85f5b10eb786c8cf31bf268131a2345a295d88d318310dc8457d831f0a587ec1600e43beb7f55aec2248483b9a95e905a468b592f0c910443b4aaa9baeb2e3,
 6.0.411, dc8aa1f84ad97cf25a979bfc243c200b7a8e73b930b68d5eed782743d88aad823c32c267c83d7a19d3c4f910a8fae7f12d07ea5a35a1d3a97e13a8674d29037b,
 6.0.412, ee97aa5258e05aecadc66e844fa8eef97d51e5035492999b974cc8272e4db1a862a1f88a925f38be9e95c71d2e961b56459dcd614475895df608945c8057c311,
+6.0.416, 5a3c60c73b68e9527406a93c9cc18941d082ac988d0b4bfea277da3465c71777dded1b3389f0dde807eda6a8186fcf68d617d2473a52203cb75127ab3dafc64d,
```

Comment:
@daniv-msft @william-msft I think adding these versions to this file will cause CI to fail in the future if we once again need to pull them from the production storage account -- unless these are the versions that include the fix for the issue we were seeing previously?

Reply:
Yeah that's annoying. We will need to remove the SDKs for these as well unfortunately.
Could you please clarify what will fail? I thought that you updated the tests to take into account this case/

Reply:
There were other tests that were updated to take this scenario into consideration, but there's still a storage account test that will try to map everything in a storage account (staging or production) to what _should have_ been generated and published to the storage account according to this `versionsToBuild.txt` file.

Reply:
Thanks @cormacpayne. So basically we need to:
1. Merge this PR
2. Remove the problematic SDKs
3. Send another small PR to update versionsToBuild.txt so that we don't block future Oryx-CI pipelines.

Does it sound right?

Reply:
I _think_ the exact ordering is:

(1) Merge the PR
(2) Publish the SDKs to the production storage account
(3) Run the Oryx CI pipeline to ensure the runtimes are published to MCR
(4) Pull the SDKs from production storage account
(5) Send out a PR to update `versionsToBuild.txt` to remove these versions so future Oryx CI pipelines can succeed